sktime/networks/cnn.py

```diff
@@ -22,6 +22,13 @@ class CNNNetwork(BaseDeepNetwork):
     filter_sizes    : array of int, shape = (nb_conv_layers)
     activation      : string, default = sigmoid
         keras activation function
+    padding         : string, default = "valid"
+        The padding type for the convolutional layers.
+
+        - 'valid': No padding is applied. The input is not padded, and the output size is reduced.
+        - 'same': The input is padded with zeros so that the output size is the same as the input size.
+
+        Note: This parameter determines how the input is padded or not padded during the convolution operation. It affects the output size of the convolution layers.
```

Comment:
This is probably too verbose, isn't it? One of the additional values a user can have is "causal" (see the padding argument section in [Conv1D](https://www.tensorflow.org/api_docs/python/tf/keras/layers/Conv1D) for reference).

We'd probably be better off mentioning that we allow all values as allowed by TensorFlow's `Conv1D`, so we won't be required to update this docstring if they choose to add/remove values.

Reply:
Thank you for the respond, I forgot the `casual` is even there. Anyway, I will add it and make the whole thing more concise.

Reply:
I think you haven't added it, I updated the dostring.